src/main.tsx

```diff
@@ -2,11 +2,37 @@ import { StrictMode } from 'react';
 import { createRoot } from 'react-dom/client';
 
 import App from './App';
+import {
+  CardExpirationProvider,
+  CardNumberProvider,
+  CardOwnerNameProvider,
+  CardPasswordProvider,
+  CardSecretCodeProvider,
+  CardSelectModalProvider,
+  CardAliasProvider,
+} from './context';
+import { CardListProvider } from './context/CardList';
 
 import './index.css';
 
 createRoot(document.getElementById('root') as HTMLElement).render(
   <StrictMode>
-    <App />
+    <CardSelectModalProvider>
+      <CardExpirationProvider>
+        <CardNumberProvider>
+          <CardOwnerNameProvider>
+            <CardSecretCodeProvider>
+              <CardPasswordProvider>
+                <CardAliasProvider>
+                  <CardListProvider>
+                    <App />
+                  </CardListProvider>
+                </CardAliasProvider>
+              </CardPasswordProvider>
+            </CardSecretCodeProvider>
+          </CardOwnerNameProvider>
+        </CardNumberProvider>
+      </CardExpirationProvider>
+    </CardSelectModalProvider>
```

Comment:
```suggestion
    <CardProviders>
      <App />
    </CardProviders>
```

앱에 추가적으로 라이브러리 Provider를 주입하는 경우도 많아서 저는 미리 분리해두는편이에요..!

Provider가 많은 부분은 다른 코멘트에서 질문드려서 패스할게요 😅

Reply:
분리해두면 장점이 하나 더 있을 것 같은데..
storybook에서 Context api를 구독할 때 Context를 주입해주기 편할거에요.

e.g.
```jsx
const withCardProvider = (children: ReactNode) => <CardProvider>{children}</CardProvider>
const Template: ComponentStory<typeof CardNumberFields> = () => withCardProvider(<CardNumberFields />);
```

Reply:
이 생각을 못했네요!
저도 분리해두는게 훨씬 좋을 것 같습니다!